src/axom/mir/EquiZAlgorithm.hpp

```diff
@@ -367,37 +356,28 @@ class EquiZAlgorithm : public axom::mir::MIRAlgorithm
     n_tmpInput[n_coordset.path()].set_external(n_coordset);
     n_tmpInput[n_fields.path()].set_external(n_fields);
     n_tmpInput[n_matset.path()].set_external(n_matset);
-    conduit::relay::io::blueprint::save_mesh(n_tmpInput,
-                                             "debug_equiz_input",
-                                             "hdf5");
+    conduit::relay::io::blueprint::save_mesh(n_tmpInput, "debug_equiz_input", "hdf5");
 #endif
 
 #if defined(AXOM_EQUIZ_SPLIT_PROCESSING)
     // _mir_utilities_zlb_begin
```

Comment:
I wonder what impact the longer lines has on source code that gets included into the Sphinx documentation. Do you think it will wrap there if the lines are too long for the Sphinx format?

Reply:
One way to find out -- https://axom.readthedocs.io/en/feature-kweiss-clang-format-updates/

Looks pretty reasonable to me. Some fields have a horizontal scrollbar:
![image](https://github.com/user-attachments/assets/693dd55f-40de-49b0-8604-78abf56270ce)

Reply:
That's good it adds the scrollbar, if needed.

Reply:
Looks like we can add custom css to modify the column width

Adding this:
```css
.wy-nav-content {
    max-width: 90%; /* Increase the width to 90% of the viewport */
    width: 940px; /* Set a fixed width (optional) */
}
```
updates the page to the following:
![image](https://github.com/user-attachments/assets/6f53a98a-5b01-4493-ac09-b2c20f7b540d)

Reply:
I'll add the change to the rtd css to this branch.

Reply:
Done in https://github.com/LLNL/axom/pull/1531/commits/44baef5ae11a37ccf7af419bf4a15bf4aec44b9c